source/reference/command/replSetReconfig.txt

```diff
@@ -53,15 +53,28 @@ considerations:
 - A majority of the set's members must be operational for the
   changes to propagate properly.
 
-- This command can cause downtime as the set renegotiates
-  primary-status. Typically this is 10-20 seconds, but could
-  be as long as a minute or more. Therefore, you should attempt
-  to reconfigure only during scheduled maintenance periods.
-
-- In some cases, :dbcommand:`replSetReconfig` forces the current
-  primary to step down, initiating an election for primary among the
-  members of the replica set. When this happens, the primary node will drop
-  all current connections.
+- The :method:`rs.reconfig()` shell method can trigger the current 
+  primary to step down in some situations. When the primary steps down, 
+  it forcibly closes all client connections.
+  
+  Primary step-down triggers an :ref:`election <replica-set-elections>` 
+  to select a new :term:`primary`. The median time in which a cluster 
+  may operate without a primary should not typically exceed 11.5 
```

Comment:
"in which a cluster may operate without a primary"  makes it sound like the replica set  can still work without a primary for about 11.5 seconds but after 11.5 seconds, well...